bears/vcs/git/GitCommitBear.py

```diff
@@ -25,6 +25,16 @@ class GitCommitBear(GlobalBear):
     LICENSE = 'AGPL-3.0'
     ASCIINEMA_URL = 'https://asciinema.org/a/e146c9739ojhr8396wedsvf0d'
     CAN_DETECT = {'Formatting'}
+    ISSUE_INFO = {
+        'github': {
+                    'issue': r'(?:\w+/\w+)?#(\S+)',
```

Comment:
`'issue'` should be indented four spaces from `'github'`, i.e. appearing under `hub..`